packages/app-store/templates/audit-log-implementation/components/ManagedAuditLogEventDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
```

Comment:
Event disabling/enabling is done per event only. This Dialog confirms user desired action before executing it. Kinda like how when you want to delete a repository github makes you write down its name. This is much simpler, only a button click, but still nice UX.